jsonnet/thanos-ruler.libsonnet

```diff
@@ -1,457 +1,458 @@
 // TODO(paulfantom): This should use upstream kube-thanos project.
+local thanosRule = import 'kube-thanos/kube-thanos-rule.libsonnet';
 
-function(params) {
-  local cfg = params,
-
-  mixin:: (import 'github.com/thanos-io/thanos/mixin/alerts/rule.libsonnet') {
-    rule+:: {
-      selector: 'job="thanos-ruler"',
+function(params)
+  local tr = thanosRule(params);
+  local cfg = tr.config;
+  tr {
+    mixin:: (import 'github.com/thanos-io/thanos/mixin/alerts/rule.libsonnet') {
+      rule+:: {
+        selector: 'job="thanos-ruler"',
+      },
     },
-  },
 
-  thanosRulerPrometheusRule: {
-    apiVersion: 'monitoring.coreos.com/v1',
-    kind: 'PrometheusRule',
-    metadata: {
-      name: 'thanos-ruler',
-      namespace: 'openshift-user-workload-monitoring',
+    thanosRulerPrometheusRule: {
+      apiVersion: 'monitoring.coreos.com/v1',
+      kind: 'PrometheusRule',
+      metadata: {
+        name: 'thanos-ruler',
+        namespace: 'openshift-user-workload-monitoring',
+      },
+      spec: $.mixin.prometheusAlerts,
     },
-    spec: $.mixin.prometheusAlerts,
-  },
 
-  trustedCaBundle: {
-    apiVersion: 'v1',
-    kind: 'ConfigMap',
-    metadata: {
-      name: 'thanos-ruler-trusted-ca-bundle',
-      namespace: cfg.namespace,
-      labels: {
-        'config.openshift.io/inject-trusted-cabundle': 'true',
+    trustedCaBundle: {
+      apiVersion: 'v1',
+      kind: 'ConfigMap',
+      metadata: {
+        name: 'thanos-ruler-trusted-ca-bundle',
+        namespace: cfg.namespace,
+        labels: {
+          'config.openshift.io/inject-trusted-cabundle': 'true',
+        },
```

Comment:
we should also add the commonLabels: https://github.com/thanos-io/kube-thanos/blob/main/jsonnet/kube-thanos/kube-thanos-rule.libsonnet#L29-L34. They should be available as `cfg.commonLabels`.